feature-flags.js

```diff
@@ -72,16 +81,24 @@ class FeatureFlagsPoller {
         }
         this.poller = setTimeout(() => this._loadFeatureFlags(), this.pollingInterval)
 
-        const res = await this._request({ path: 'api/feature_flag', usePersonalApiKey: true })
+        try {
+            const res = await this._request({ path: 'api/feature_flag', usePersonalApiKey: true })
+            if (res && res.status === 401) {
+                throw new ClientError(
+                    `Your personalApiKey is invalid. Are you sure you're not using your Project API key? More information: https://posthog.com/docs/api/overview`
+                )
+            }
 
-        if (res && res.status === 401) {
-            throw new Error(
-                `Your personalApiKey is invalid. Are you sure you're not using your Project API key? More information: https://posthog.com/docs/api/overview`
-            )
-        }
+            this.featureFlags = res.data.results.filter(flag => flag.active)
 
-        this.featureFlags = res.data.results.filter(flag => flag.active)
-        this.loadedSuccessfullyOnce = true
+            this.loadedSuccessfullyOnce = true
+        } catch (err) {
+            // if an error that is not an instance of ClientError is thrown
+            // we silently ignore the error when reloading feature flags
+            if (err instanceof ClientError) {
+                throw err
```

Comment:
can we log the error if debug is set? We can add that in as a new option to `init` or even just read `DEBUG` from the env